ubuntu_image/tests/test_parser.py

```diff
@@ -308,37 +309,7 @@ def test_size_offset_suffix(self):
         partition0 = volume0.structures[0]
         self.assertEqual(partition0.size, MiB(3))
 
-    def test_mbr_structure(self):
```

Comment:
I removed this test as I added earlier the test_volume_structure_role that already tested the role: mbr part - but yeah, since now we need to support both the new and the old, I'll re-introduce this test but maybe with a different name.